From 26abbe4af25eeb3737f40dcb04cfb1c22805ed0b Mon Sep 17 00:00:00 2001
From: Kirill Kapranov <kirill.kapranov@compulab.co.il>
Date: Mon, 3 Dec 2018 19:17:35 +0200
Subject: [PATCH 95/96] ucm-imx8: dts: Add HDMI support

Add the alternative dtb 'sbc-ucm-imx8-hdmi' with a single display support
over HDMI.

Signed-off-by: Kirill Kapranov <kirill.kapranov@compulab.co.il>
---
 arch/arm64/boot/dts/compulab/Makefile              |  1 +
 arch/arm64/boot/dts/compulab/sbc-ucm-imx8-hdmi.dts | 36 ++++++++++++++++++++++
 2 files changed, 37 insertions(+)
 create mode 100644 arch/arm64/boot/dts/compulab/sbc-ucm-imx8-hdmi.dts

diff --git a/arch/arm64/boot/dts/compulab/Makefile b/arch/arm64/boot/dts/compulab/Makefile
index a37f887..4cc880d 100644
--- a/arch/arm64/boot/dts/compulab/Makefile
+++ b/arch/arm64/boot/dts/compulab/Makefile
@@ -3,6 +3,7 @@ dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8-lvds-dcss.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-imx8-dual.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += ucm-imx8.dtb
+dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-ucm-imx8-hdmi.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-ucm-imx8.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-ucm-imx8-sd-card.dtb
 dtb-$(CONFIG_ARCH_FSL_IMX8MQ) += sbc-ucm-imx8-lvds-dcss.dtb
diff --git a/arch/arm64/boot/dts/compulab/sbc-ucm-imx8-hdmi.dts b/arch/arm64/boot/dts/compulab/sbc-ucm-imx8-hdmi.dts
new file mode 100644
index 0000000..3c6927b
--- /dev/null
+++ b/arch/arm64/boot/dts/compulab/sbc-ucm-imx8-hdmi.dts
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2018 CompuLab Ltd.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ */
+
+/dts-v1/;
+
+#include "ucm-imx8.dtsi"
+#include "sb-ucm-imx8.dtsi"
+#include "ucm-imx8-wifi.dtsi"
+#include "sb-ucm-imx8-lvds-lcdif.dtsi"
+
+/ {
+	model = "CompuLab UCM-iMX8 on SBC-UCM-iMX8";
+	compatible = "compulab,sbc-ucm-imx8", "compulab,ucm-imx8", "fsl,imx8mq";
+};
+
+&dcss {
+	disp-dev = "hdmi_disp";
+	status = "okay";
+};
+&lcdif {
+	status = "disabled";
+};
+&hdmi {
+	status = "okay";
+};
-- 
1.9.1

